docs/api_requests/block-vhost-user.md

```diff
@@ -74,6 +74,28 @@ be coming from the fact that the custom logic is implemented in the same process
 that handles Virtio queues, which reduces the number of required context
 switches.
 
+## Disadvantages
+
+In order for the backend to be able to process virtio requests, guest memory
+needs to be shared by the frontend to the backend. This means, a shared memory
+mapping is required to back guest memory. When a vhost-user device is
+configured, Firecracker uses `memfd_create` instead of creating an anonymous
+private mapping to achieve that. It was observed that page faults to a shared
+memory mapping take significantly longer (up to 24% in our testing), because
+Linux memory subsystem has to use atomic memory operations to update page
+status, which is an expensive operation under specific conditions. We advise
+users to profile performance on their workloads when considering to make use of
+vhost-user devices.
+
+## Other considerations
+
```

Comment:
nit: I think this paragraph is convoluted. can be simplified in something like: "vhost-user block device, use a customer-implemented backend instead of drive file in the host managed by the host OS, as in the virtio case. The host OS file implementation benefits from caching file access in the host. Therefore, according workload performance needs, it may beneficial to implement caching mechanism in the vhost-user block backed. " or something similar.

Reply:
I compared your version sentence by sentence with the original and couldn't find major differences...

Reply:
is there something specific you'd like to improve in the paragraph?

Reply:
the text is overall different, it may be subjective but i think does not read fluently. But happy to skip if you think the version proposed is not an improvement.